decred_wallet/SecurityMenuViewController.swift

```diff
@@ -246,6 +262,21 @@ class SecurityMenuViewController: UIViewController,UITextFieldDelegate {
             self.present(alert, animated: true, completion: nil)
         }
     }
+    func toggleView(){
```

Comment:
Instead of hiding each view, how about putting them all in a container and toggling the visibility of the container?